packages/cli/src/lingui-compile.ts

```diff
@@ -68,14 +79,14 @@ function command(config: LinguiConfig, options) {
 
           if (options.verbose) {
             console.error(chalk.red("Missing translations:"))
-            missingMsgIds.forEach((msgId) => console.log(msgId))
+            missingMsgIds.forEach((msgId) => console.error(msgId))
           } else {
             console.error(
               chalk.red(`Missing ${missingMsgIds.length} translation(s)`)
             )
           }
           console.error()
-          process.exit(1)
+          return false
```

Comment:
will it be correctly handled and exit with the `1` code as a result?

Reply:
Yes, when command return false the orchestrator script would return 1 (it's below in the code)